src/examples/stripe-bots/README.md

```diff
@@ -0,0 +1,56 @@
+# Stripe and Medplum
+
+Many applications use Stripe to process payments, and keeping records of payments in the context of patient records can help streamline your workflow.
+
+At a high level, the integration has the following components:
+
+1. The user initiates a payment on a custom web application or using a [Stripe Payment Link](https://stripe.com/payments/payment-links).
+2. A webhook, containing details of the transaction as a [Stripe Event Object](https://stripe.com/docs/api/events/object) are sent to Medplum Bot
+3. The Medplum Bot processes the `event` and creates or updates the relevant FHIR resources, e.g. [Invoice](https://www.medplum.com/docs/api/fhir/resources/invoice)
+
+## Medplum Setup
+
+Create your [Medplum Access Policy](https://www.medplum.com/docs/auth/access-control#resource-type), you'll want to include three resource types `Invoice`, `Account` and `Bot` (read-only). Example below.
```

Comment:
Some explanation of _why_ an `AccessPolicy` is best practice here would be useful.  E.g

> Because we are granting access to a 3rd party service, we'll want to create an access policy to limit the ability to write resources